yjit/src/stats.rs

```diff
@@ -165,6 +165,8 @@ make_counters! {
     send_keywords,
     send_kw_splat,
     send_args_splat,
```

Comment:
Is the `send_args_splat` counter still used?

Reply:
Removed thanks